### PR TITLE
Loading spinner issue on SignIn Page

### PR DESCRIPTION
**Pull request by New-Bee-Star**
This PR fix https://github.com/opencollective/opencollective/issues/1688
Changed InitialState regarding to the token value

### CODE REVIEW

**Comment by znarf** [comment]
Thank you for that! As you might have seen, an automated deployment was done using your branch: https://opencollective-frontend-jprsicwvtq.now.sh/

I tested and have 2 important feedback after that:
- When a (Sign In) token is being passed, the spinner is the right thing to display, not the form. ie: https://opencollective-frontend-jprsicwvtq.now.sh/signin/{ACCESS_TOKEN} In the top bar "Sign In" should be replaced by the "Sign In" spinner.
- When you are already authenticated and loading any page, when the Client side authentication kicks in, in the top bar "Sign In" should be replaced by the "Sign In" spinner.

Check https://staging.opencollective.com/ behavior to compare with the branch.

**Comment by znarf** [comment]
Also, please notice that CI is not passing: https://circleci.com/gh/opencollective/opencollective-frontend/4776 Maybe @Betree can help?

**Comment by New-Bee-Star** [comment]
I will handle this issue by myself.

File: src/pages/signin.js [hunk 1]
```diff
@@ -69,7 +68,8 @@ class SigninPage extends React.Component {
   }
 
   renderContent() {
-    if (this.props.loadingLoggedInUser || this.state.success) {
+    console.log(this.props.token, this.props.loadingLoggedInUser, this.state.success);
```
Comment (znarf) [inline src/pages/signin.js]: We don't want a console.log there.

**Comment by znarf** [comment]
@New-Bee-Star can you update the title and description of the PR?

**Comment by znarf** [comment]
@New-Bee-Star are you able to make this PR just 1 commit? and can you write a commit message that explains what it's doing and what it's fixing? See: https://github.com/opencollective/opencollective-frontend/blob/master/CONTRIBUTING.md

Let me know if that's a problem, we can always "Squash and merge".

**Comment by znarf** [comment]
(Got private feedback from @New-Bee-Star to "Squash and merge".

**Pull request merged**